community/modules/remote-desktop/chrome-remote-desktop/variables.tf

```diff
@@ -65,6 +69,16 @@ variable "instance_image" {
     family  = "debian-12"
     project = "debian-cloud"
   }
+
+  validation {
+    condition     = contains(["ubuntu-2204-lts", "debian-12"], var.instance_image.family)
+    error_message = "In var.instance_image, the 'family' field must be 'ubuntu-2204-lts' or 'debian-12'."
+  }
+
+  validation {
+    condition     = contains(["ubuntu-os-cloud", "debian-cloud"], var.instance_image.project)
+    error_message = "In var.instance_image, the 'project' field must be 'ubuntu-os-cloud' or 'debian-cloud'."
```

Comment:
These validations are overly restrictive. What if I want to build a custom image based off these images. Also, I believe this will break the slurm use case which uses a slurm image.

I think the best we can say is that "if you use project x, then you must use family y".